src/js/stores/AppFormStore.js

```diff
@@ -295,16 +295,15 @@ function processResponseErrors(responseErrors, response, statusCode) {
       AppFormErrorMessages.getGeneralMessage("unknownServerError");
 
   } else if (statusCode === 422 && response != null &&
-      Util.isArray(response.errors)) {
-
-    response.errors.forEach((error) => {
-      var attributePath = error.attribute.length
-        ? error.attribute
+      Util.isArray(response.details)) {
```

Comment:
Mmmh OK, I thought this was pretty clear
> The changes introduced in https://github.com/mesosphere/marathon/pull/2975/files#diff-7bb478205fd47f472265c276e29b7cfaR227 broke our 422 error parsing scripts. 

Also, the linked issue had a lot of descriptive info, I thought. Anyway, sorry if that caused you trouble!